inst/css/filter-panel.css

```diff
@@ -151,11 +151,19 @@ a.remove_all:hover {
 }
 
 .filterPlotOverlayRange {
-  margin: 10px 11px -25px 11px;
+  margin: 10px 0px -25px;
   height: 25px;
   animation: 0.75s ease-out 0s 1 shinyDataFilterEnlargeY,
     0.5s ease-in 0s 1 shinyDataFilterFadeIn;
   transform-origin: bottom;
+  width: 100%;
+  max-width: 100%;
+}
+
+/* used to ensure the range histogram exactly matches the slider */
+.filterRangeSlider .irs-grid {
+  width:100% !important;
+  left:0% !important;
```

Comment:
I noticed the draggable circle position is not matching with the grid.

<img src="https://user-images.githubusercontent.com/8597300/211120279-c19b4f06-1625-4123-86c0-fa40e1634a81.jpg" width="10%" height="10%" />

How about we update the `.irs-line` instead?

```css
.filterRangeSlider .irs-line {
  margin-left: 10px;
  margin-right: 10px;
}
```

The circle is more centered to the value on the grid:
<img src="https://user-images.githubusercontent.com/8597300/211120786-c42b619f-f627-45ce-b786-dccbab6c7ad5.jpg" width="10%" height="10%" />

And our grid still look okay.
<img src="https://user-images.githubusercontent.com/8597300/211120803-208a8543-3901-420d-8db4-b8a07334f044.jpg" width="50%" height="50%" />

Reply:
yup that's nicer

Reply:
ok so I see the problem as well but replacing the .irs-grid with .irs-line makes the graph be too big for me - this is bs3
![image](https://user-images.githubusercontent.com/15201933/211521429-6789344e-9129-410c-8328-caad95296ea4.png)

Reply:
I wonder if we need our own widget here...

Reply:
To make things weider, swapping `.irs-grid` with `.irs-line` looked fine for me on all BS versions.